Closes #56: Implemented the java bean for profile

Qw3rty26 opened the pull request:
Closes #56, a new Java Class called `ProfileBean.java` was added inside the `/src/main/java` directory.

It implements the `java.io.Serializable` class and has everything needed to store the "name" and "password" attributes for the profile.

The constructor of the bean sets "name" and "password" to `null` by default, it is then needed to use the corrispective 
`void: setName(String: name)` and `void: setPassword(String: password)` 
methods to populate the bean.

After the Bean is successfully created and instantiated, we can then get "name" and "password" with the methods:

`String: getName(void)` and `String: getPassword(void)`



`Login.java` class has also been updated to correctly use `ProfileBean.java`.

Qw3rty26 committed:
Closes #56: Implemented the java bean for profile

San7o commented on src/main/java/ProfileBean.java:
```diff
@@ -0,0 +1,40 @@
+                name = null;
+                password = null;
+        }
+
```
You should add the constructor
```java
ProfileBean(String name, String password){
    this.name = name;
    this.password = password;
}
```

San7o reviewed (changes requested):
Add a method, everything else looks good

San7o commented on src/main/java/ProfileBean.java:
```diff
@@ -0,0 +1,40 @@
+//(such as "name" or "password") within the webapp, whilst keeping it
+//MVC compliant and Thread safe.
+
+public class ProfileBean implements Serializable {
```
Change `ProfileBean` to `TentativcAcessoBean`

San7o commented on src/main/java/Login.java:
```diff
@@ -81,20 +81,23 @@ protected void doPost (HttpServletRequest req,
-        String name = req.getParameter("name");
-        String password = req.getParameter("password");
+        //Create Java Bean for profile
+	ProfileBean profile = new ProfileBean();
```
change `ProfileBean` to `TentativoAccessoBean`

San7o reviewed (changes requested):
Change the name of the class

San7o commented:
I need this code fixes asap, I'll close the PR. This, with the fixes, ill be implemented in my next PR

Qw3rty26 commented:
@San7o I understand your immediate need of this Pull Request to be added to our webapp, however please note you can't just delete done work just like that, instead please be a bit more patient and wait for changes to be made.

Thank you.

Qw3rty26 committed:
Fixes #56: Changed the bean class name and added another constructor

Qw3rty26 commented:
I updated the code for the Java Bean, its name is now `TentativoAccessoBean.java` as requested.
A new constructor was also added.
Everything should be looking good now.

Qw3rty26 commented:
this pull request isnt needed anymore after #72 so i guess it will be sent into the trash
